components/server/src/orgs/organization-service.ts

```diff
@@ -598,33 +599,12 @@ export class OrganizationService {
             }
 
             if (settings.onboardingSettings.welcomeMessage) {
-                const welcomeMessage = settings.onboardingSettings.welcomeMessage;
-
-                if (welcomeMessage.featuredMemberResolvedAvatarUrl) {
+                if (settings.onboardingSettings.welcomeMessage.featuredMemberResolvedAvatarUrl) {
                     throw new ApplicationError(
                         ErrorCodes.BAD_REQUEST,
                         "featuredMemberResolvedAvatarUrl is not allowed to be set",
                     );
                 }
-
```

Comment:
It's ok to extract the code, but it should still be called here, not anywhere else: This is the one-and-only spot in our code base where we do domain-level validations. Also helps writing tests for it. :slightly_smiling_face:

Reply:
As we discussed yesterday, I will be removing this check entirely and just leave the frontend one in to reduce complexity, given this is just trying to help the UX.

Reply:
Addressed in [`d9e1cf2` (#20626)](https://github.com/gitpod-io/gitpod/pull/20626/commits/d9e1cf2d375af659a24324f6e360bcd44f1affd9)

Reply:
> As we discussed yesterday, I will be removing this check entirely and just leave the frontend one in to reduce complexity, given this is just trying to help the UX.

Ah, see it now. Thanks!